src/plugins/maps_legacy/public/map/service_settings.js

```diff
@@ -27,10 +27,10 @@ import { ORIGIN } from '../common/constants/origin';
 const TMS_IN_YML_ID = 'TMS in config/kibana.yml';
 
 export class ServiceSettings {
-  constructor() {
+  constructor(mapConfig, tilemapsConfig) {
     const getInjectedVar = getInjectedVarFunc();
-    this.mapConfig = getInjectedVar('mapConfig');
-    this.tilemapsConfig = getInjectedVar('tilemapsConfig');
+    this.mapConfig = mapConfig;
```

Comment:
I missed this earlier, but do you mind renaming these to `_mapConfig` and `_tilemapsConfig` to be consistent. Private-vars are preceded by `_`.